### ISSUE TITLE
[leo_object_storage] Development Branch Travis CI failing

### ISSUE THREAD

**Issue by windkit** [leo-project/leofs#542]
## Description
It is a old issue has been happening for 2 years, Travis CI is failing because the output to stdout exceeds the 4MB limit.

1.4 Branch does not have problem though

## Details
The logging of worker state includes the whole binary of objects, therefore emitting lots of log.
```
=ERROR REPORT==== 29-Jun-2015::07:26:40 ===
** State machine leo_compact_worker_0 terminating 
** Last event in was {compaction_event_info,undefined,run,undefined,
                         undefined,false,false,undefined}
** When State == running
**      Data  == {compaction_state,leo_compact_worker_0,leo_object_storage_0,
                     leo_metadata_0,
                     {backend_info,undefined,<<"LeoFS AVS-2.4">>,
                         <<"LeoFS AVS-2.4">>,
                         "/home/travis/build/leo-project/leo_object_storage/.eunit/comaction_test/object/0.avs",
                         "/home/travis/build/leo-project/leo_object_storage/.eunit/comaction_test/object/0.avs_63602781943",
                         {file_descriptor,prim_file,{#Port<0.44083>,21}},
                         {file_descriptor,prim_file,{#Port<0.44085>,22}}},
                     <0.393.0>,leo_diagnosis_log_0,running,true,false,false,
                     119,300,39,40,150,10,
                     {compaction_prms,<<"TEST_249">>,
                         <<14,74,14,206,65,98,60,141,179,134,255,206,9,156,63,

...
```
https://travis-ci.org/leo-project/leo_object_storage/builds/68766552

**Comment by yosukehara** [comment]
I've checked this issue, and then found the reason of the termination:

```erlang
** Reason for termination = 
** {{{case_clause,ok},
     [{leo_object_storage_server,open_container,1,
                                 [{file,"src/leo_object_storage_server.erl"},
                                  {line,676}]},
      {leo_object_storage_server,handle_call,3,
                                 [{file,"src/leo_object_storage_server.erl"},
                                  {line,593}]},
      {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,580}]},
      {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},
    {gen_server,call,
                [leo_object_storage_0,
                 {switch_container,"/home/travis/build/leo-project/leo_object_storage/.eunit/comaction_test/object/0.avs_63602781943",
                                   151,81477180},
                 infinity]}}
```

**Comment by mocchira** [comment]
@windkit thank you for pointing this out. 
I totally missed.

@yosukehara I might be able to fix this issue (now UT is WIP
it may be kind of a race condition bug only happened under a resource constrained environment.

**Comment by mocchira** [comment]
@windkit @yosukehara 
https://github.com/leo-project/leo_object_storage/commit/2225db71b137f11eac23d7d3bab45ff85719d754
fixed this issue. ( but UT still failed due to another problem )
please check it out on your env too.

**Comment by mocchira** [comment]
now the below error is happening (file:pread return einval. I have not ever seen before)
https://travis-ci.org/#L4980

**Comment by windkit** [comment]
I can reproduce the situation when I use 17.1.

Unit Tests can complete with 17.5 (even before the fix)

**Comment by yosukehara** [comment]
FIXED with v1.3.1
- https://travis-ci.org/leo-project/leo_object_storage
